overlord/snapstate/snapstate.go

```diff
@@ -485,6 +485,12 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 		prev = copyData
 	}
 
+	if !release.OnClassic {
+		createSnapSave := st.NewTask("create-snap-save", fmt.Sprintf(i18n.G("Create snap %q save data directory"), snapsup.InstanceName()))
```

Comment:
Is the text `Create snap %q save data directory` fine here? I keep feeling like maybe the word `directory` is a bit offending here, but I have a hard time formalizing the sentence. Or maybe I'm just overthinking it and it's fine.